### ISSUE TITLE
[QUESTION] Advantages compare to existing alternates such as lsd and eza

### ISSUE THREAD

**Issue by CarterLi** [Equationzhao/g#166]
I'm an eza user and new to this project. It would be awesome to see some feature and performance comparation between eza and g-ls. Thanks.

**Comment by Equationzhao** [comment]
😂 since you've been using eza, I'll recommend you sticking with it. 

`eza` can provide better performance than `g` up to 3x on macOS.
(as I tested, `eza` will take 5ms/120ms for 20/10000 files, and `g` will take 10ms/400ms, FYI)

well, `g` does have some unique features like 
1. listing files' mime-type/charset
2. listing files' modified, created, accessed time together (`eza` seems can only list one of those field once a time)
3. listing files' in table/markdown/json/csv/tsv view
4. show files' relative path to some dir
5. customizable theme (not sure if `eza` has similar settings 🤔, but `g` does provide the ability to customize your theme)

I started the project as a way to practice my Go skills, and definitely it has some weakness like the lack of completion(only zsh is supported yet), lack of test. But I'll try to improve the project~


btw, I've been using fastfetch for a long time, thanks for your awesome project 🥳
